x-pack/plugins/ux/public/hooks/use_client_metrics_query.ts

```diff
@@ -35,11 +35,11 @@ export function useClientMetricsQuery() {
   );
 
   const data = useMemo(() => {
-    if (!esQueryResponse) return {};
+    if (!esQueryResponse?.aggregations) return {};
 
     const {
       hasFetchStartField: { backEnd, totalPageLoadDuration },
-    } = esQueryResponse.aggregations!;
+    } = esQueryResponse.aggregations;
```

Comment:
This is a good scenario to unit test, I'll try to create a separate PR for that.